.bazelrc

```diff
@@ -44,9 +37,6 @@ build:avx_linux --host_copt=-mavx
 build:native_arch_posix --copt=-march=native
 build:native_arch_posix --host_copt=-march=native
 
-build:mkl_open_source_only --define=tensorflow_mkldnn_contraction_kernel=1
-
-build:cuda --repo_env TF_NEED_CUDA=1
```

Comment:
Some of these options are still in use in openxla/xla, e.g. https://github.com/openxla/xla/blob/ab3e2de65f3c077e4ea7b4d474dd2453e54e9c8e/.bazelrc#L228

Feel free to remove any options here that don't exist in openxla. However, I think we should leave in anything they still use to be safe. Their build is very complicated and nobody really understands it, so I'm inclined to play it safe. @stgpetrovic what do you think?